torch/csrc/autograd/function.h

```diff
@@ -563,6 +563,14 @@ struct MakeNextFunctionList : IterArgs<MakeNextFunctionList> {
       next_edges.emplace_back();
     }
   }
+  void operator()(const Variable* variable) {
```

Comment:
Not sure you need this overload; if you have a const Variable* you can always dereference it to get a const Variable&